extensions/positron-python/src/client/terminals/codeExecution/helper.ts

```diff
@@ -95,8 +101,11 @@ export class CodeExecutionHelper implements ICodeExecutionHelper {
             let smartSendSettingsEnabledVal = true;
             const configuration = this.serviceContainer.get<IConfigurationService>(IConfigurationService);
             if (configuration) {
-                const pythonSettings = configuration.getSettings(this.activeResourceService.getActiveResource());
-                smartSendSettingsEnabledVal = pythonSettings.REPL.enableREPLSmartSend;
+                // --- Start Positron ---
+                // This setting is hidden in favor of the Positron console.
+                // const pythonSettings = configuration.getSettings(this.activeResourceService.getActiveResource());
+                // smartSendSettingsEnabledVal = pythonSettings.REPL.enableREPLSmartSend;
```

Comment:
this one was slightly more annoying since it isn't wrapped in a nice helper function!